qdrant-landing/content/documentation/frameworks/langchain4j.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Langchain4J
+weight: 2100
+---
+
+# Langchain4J
+
+[Langchain4J](https://github.com/langchain4j/langchain4j) is a community port of [Langchain](https://www.langchain.com/) for building context-aware AI applications in Java
```

Comment:
Langchain4J is a community port. Not affiliated with https://www.langchain.com/.